cmd/execution/main.go

```diff
@@ -77,6 +79,24 @@ func main() {
 	)
 
 	cmd.FlowNode(flow.RoleExecution.String()).
+		CreateState(func(fnb *cmd.FlowNodeBuilder) (protocol.MutableState, error) {
```

Comment:
Currently only `access` node uses the `FollowerState`. I think all nodes except the `consensus` node should be using `FollowerState`. Only consensus nodes need to perform the in-depth main chain payload validation.